.travis.yml

```diff
@@ -1,8 +1,55 @@
+sudo: false
+
 language: php
-php:  
-  - 5.6
-  - 7.0
 
-before_script: composer install
+cache:
+  directories:
+    - $HOME/.composer/cache
```

Comment:
What is the purpose of this cache directories on travis ?

Reply:
prevent composer packs from being downloaded several times